features/head.feature

```diff
@@ -0,0 +1,26 @@
+@head
```

Comment:
This tag should be used to reset to the default configuration on an `After` block. See how it's done with the `@footer` tag.